operating/manage-compliance-operator/compliance-operator-rhacs.adoc

```diff
@@ -32,4 +32,4 @@ $ oc -n stackrox delete pod -lapp=sensor
 After performing these steps, run a compliance scan in {product-title-short} and ensure that `ocp4-cis` and `ocp4-cis-node` results are displayed. See "Running a compliance scan" in the "Additional resources" section for more information.
 
 .Additional resources
-* xref:../../operating/manage-compliance.adoc#run-compliance-scan_manage-compliance[Running a compliance scan in {product-title-short}]
```

Comment:
Since this is one of the first mentions of the Compliance Operator in this section, we could link to it's documentation on line 10?

Reply:
We might also want to clarify that the Compliance Operator must be installed before the sensor is up and running for the integration to work. Otherwise, sensor needs to be bounced? Do we want to call that out here, too?

@dashrews78 is that still correct?

Reply:
The mention to bounce sensor is still in the docs as that is not a new requirement.  Though I could see the argument that we may want to bring more attention to it.